mgmt/rest/rest_func_test.go

```diff
@@ -582,6 +583,12 @@ func TestPluginRestCalls(t *testing.T) {
 				cfg := getDefaultMockConfig()
 				err := cfgfile.Read("../../examples/configs/snap-config-sample.json", &cfg, MOCK_CONSTRAINTS)
 				So(err, ShouldBeNil)
+				if len(SNAP_AUTODISCOVER_PATH) == 0 {
+					if len(SNAP_PATH) != 0 {
+						SNAP_AUTODISCOVER_PATH = fmt.Sprintf("%s/plugin", SNAP_PATH)
+					}
+				}
+				cfg.Control.AutoDiscoverPath = SNAP_AUTODISCOVER_PATH
```

Comment:
In fact, the tests were first failing because in the examples/configs/snap-config-sample.json file the path is set to a dummy value "auto_discover_path": "/some/directory/with/plugins", therefore I had the choice to either ignore/warn about this error (I guess 'ignore' was the previous behaviour) either make sure that tests could run properly by using SNAP_PATH (which is taken by default if SNAP_AUTODISCOVERY_PATH and as it is set automatically by the make check command this was nice to use) or if the user running test has a separate location for plugins, then he is able overwrite this to test proper loading of his plugins. This is our case as we are building some plugins ourselves to use in Docker containers therefore requiring (for now at least) support for configurable /proc file system. 

Hope this makes sense